packages/salesforcedx-utils-vscode/src/commands/commandletExecutors.ts

```diff
@@ -123,7 +123,7 @@ export abstract class LibraryCommandletExecutor<T>
   protected cancellable: boolean = false;
   private cancelled: boolean = false;
   private readonly executionName: string;
-  private readonly logName: string;
+  protected readonly logName: string;
```

Comment:
I tucked the conflict msgs into baseDeployRetrieve, and this is the only piece of data I could use to figure out which command was being run while inside the catch block that handles the SourceConflict error.